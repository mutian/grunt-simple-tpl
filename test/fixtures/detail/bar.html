<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>{{{title}}} bar!</title>
    <link rel="stylesheet" href="{{{cdn_path}}}/c/bar.css">
</head>
<body>
</body>
<script src="{{{cdn_path}}}/j/bar.js"></script>
</html>