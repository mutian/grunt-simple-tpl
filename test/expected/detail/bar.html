<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>just a test bar!</title>
    <link rel="stylesheet" href="http://mycdn.com/www/1604/c/bar.css">
</head>
<body>
</body>
<script src="http://mycdn.com/www/1604/j/bar.js"></script>
</html>